### PR TITLE
contact.html portfolio_home.html portfolio.html portfolio_style.css

### PR DESCRIPTION
**Pull request by karenschulz**
portfolio_home.html

I design a very simple site. created a nav bar with a label with my name, aligned to the left and two links aligned to the right using flex.
the body has a little profile picture and some text and two links at the bottom then is the footer with copy rights.

contact.html

another simple design, this time the nav bar fills both sides, but the body has different sizes. I resized the profile picture.

still working on portfolio.html which proves more difficult to tame.

### CODE REVIEW

File: portfolio_style.css
Comment (Emily774) [inline portfolio_style.css:169]: You might want to change your sizes to percentages to make sure they will work with the mobile site, also I am not sure if the no-wrap element will work with mobiles without seeing the end product. You might also want to remove the bits that you have commented out if you are not using them. Overall looks great though!

File: portfolio.html
Comment (Emily774) [inline portfolio.html:6]: I didn't see a DOCTYPE but that might just be github.

File: portfolio.html
Comment (Emily774) [inline portfolio.html]: These div don't seem to have a </div>, also the </section> tag needs indenting

File: portfolio.html
Comment (Emily774) [inline portfolio.html]: there doesn't seem to be a closing div tag on line 25, I think that is supposed to be the </div> tag for line 18?

File: portfolio.html
Comment (Emily774) [inline portfolio.html]: I think this <div> needs to be a </div>

File: portfolio.html
Comment (Emily774) [inline portfolio.html]: I think the <footer> tags need to be within the body tag

File: portfolio.html
Comment (Emily774) [inline portfolio.html]: I think you need to have 'label for="NavBar" before having a class. Or else have the class element in a seperate div or p tag

**Review by Emily774 — COMMENTED** [review]
Overall looks great, just a few instances where I don't think you've remembered to close tags 👍

File: portfolio.html
Comment (karenschulz) [reply portfolio.html]: hi, do you mean the footer needs to be inside the body tag?

File: portfolio_style.css
Comment (karenschulz) [reply portfolio_style.css:169]: Hi, thanks for your comments. Good points!
I will work on the percentages next I think

Comment (karenschulz) [reply portfolio_style.css:169]: Hi Louse. I did the percentage bit and erased my comments I am not using. Thanks. When I update it you might have to go over them again (I will need your approval)

File: portfolio.html
Comment (karenschulz) [reply portfolio.html:6]: I think it was github, because it is in the original doc. Thanks.

File: portfolio.html
Comment (karenschulz) [reply portfolio.html]: I went over indentings and tidied them up. I also tidied that bit of coding.

File: portfolio.html
Comment (karenschulz) [reply portfolio.html]: I tidied this up, it should be now around line 56 in portfolio.html

File: portfolio.html
Comment (karenschulz) [reply portfolio.html]: I made the links... I guess that is what you meant?

File: portfolio.html
Comment (karenschulz) [reply portfolio.html]: I corrected that. Thanks!

File: portfolio.html
Comment (karenschulz) [reply portfolio.html]: changed the label issue.

**Comment by karenschulz** [comment]
I submitted Louises comments and corrected various issues. Tidied up the code, changed indentation, repaired strayed 
 things, closed opened tags, etc. I also added a form in contact.html, made the links available, added a mobile friendly version in css, added a hover and focus to my links, etc.

File: contact.html
Comment (mtmgoliath) [inline contact.html]: I assume that this "portfolio_home.html" is your home page?
If it is, then it should be named index.html as this is standard practice =]

File: contact.html
Comment (mtmgoliath) [inline contact.html]: shouldn't be using br. You should accomplish the styling using css instead.
so if you wanted more space above the p tag then you could add padding-top to it in css

File: contact.html
Comment (mtmgoliath) [inline contact.html]: replace br with css styling

File: contact.html
Comment (mtmgoliath) [inline contact.html]: i am not sure if p tags should be using inside of a form. I believe you should use divs here instead as these are not paragraphs

File: portfolio.html
Comment (mtmgoliath) [inline portfolio.html]: I dont think this image needs to nested in a p tag. just image would work i believe

File: portfolio.html
Comment (mtmgoliath) [inline portfolio.html]: for the alt info here and above please give more literal description of what the picture shows =]

File: portfolio_home.html
Comment (mtmgoliath) [inline portfolio_home.html]: in alt describe your profile pic, hair style, accessories, mood etc

File: portfolio_style.css
Comment (mtmgoliath) [inline portfolio_style.css]: better to use 10px here instead of pt

and then you can go on to use rem for rest of the sizing on the page

File: portfolio_style.css
Comment (mtmgoliath) [inline portfolio_style.css]: should use rem here. not sure how pt converts to px but if it was 50px here you would put 5rem instead.

File: portfolio_style.css
Comment (mtmgoliath) [inline portfolio_style.css:59]: use rem (5rem for these 2)

File: portfolio_style.css
Comment (mtmgoliath) [inline portfolio_style.css]: use rem:
3rem;
20rem;

File: portfolio_style.css
Comment (mtmgoliath) [inline portfolio_style.css]: use rem

File: portfolio_style.css
Comment (mtmgoliath) [inline portfolio_style.css]: rem

File: portfolio_style.css
Comment (mtmgoliath) [inline portfolio_style.css:121]: 5rem;
5rem;
10rem;

File: portfolio_style.css
Comment (mtmgoliath) [inline portfolio_style.css]: 1rem

File: portfolio_style.css
Comment (mtmgoliath) [inline portfolio_style.css]: 5rem;
5rem;
10rem;

File: portfolio_style.css
Comment (mtmgoliath) [inline portfolio_style.css]: rem

File: portfolio_style.css
Comment (mtmgoliath) [inline portfolio_style.css:149]: 1rem

File: portfolio_style.css
Comment (mtmgoliath) [inline portfolio_style.css:161]: 15rem

File: portfolio_style.css
Comment (mtmgoliath) [inline portfolio_style.css:165]: 22rem

File: portfolio_style.css
Comment (mtmgoliath) [inline portfolio_style.css]: too much indentation here

**Review by mtmgoliath — CHANGES_REQUESTED** [review]
great stuff.

few minor things: like using rem instead of pt or px

also your home page should be called index.html

File: contact.html
Comment (karenschulz) [reply contact.html]: Yes, thanks Mike. I changed it to index.html as you suggested. You are right.

File: portfolio.html
Comment (karenschulz) [reply portfolio.html]: I erased the <p> tag and you are right, it was OK.

File: portfolio.html
Comment (karenschulz) [reply portfolio.html]: I added a longer description to the alt

File: portfolio_home.html
Comment (karenschulz) [reply portfolio_home.html]: oh... really? it's quite funny! But I did what you suggested

File: portfolio_style.css
Comment (karenschulz) [reply portfolio_style.css]: Ok, you are right. Changed it

File: portfolio_style.css
Comment (karenschulz) [reply portfolio_style.css:59]: sorry, I am confused... can we talk about this? ;)
I thought that rem was for font size only...

File: portfolio_style.css
Comment (karenschulz) [reply portfolio_style.css]: changed it to rem

File: portfolio_style.css
Comment (karenschulz) [reply portfolio_style.css]: done

File: portfolio_style.css
Comment (karenschulz) [reply portfolio_style.css]: done.

File: contact.html
Comment (karenschulz) [reply contact.html]: br erased

**Review by Emily774 — APPROVED** [review]
Hi, I think you've implemented all the changes I wanted you to make so you just need to do the ones Mike suggested. Good work! It looks great!

File: contact.html
Comment (karenschulz) [reply contact.html]: and followed your advise of using div, and styled them in my CSS. Thanks

File: contact.html
Comment (karenschulz) [reply contact.html]: done!

File: contact.html
Comment (karenschulz) [reply contact.html]: I asked Rayna and she said it is possible to use<p> inside a form. You cannot use <p> whit <h1> for example. So I kept the <p> tags

File: portfolio_style.css
Comment (karenschulz) [reply portfolio_style.css]: I changed all fonts to rem as you suggested. Good one!

File: portfolio_style.css
Comment (karenschulz) [reply portfolio_style.css:59]: Ok, I talked to Rayna and she said rem is used for fonts and paddings are OK in px. Cheers

File: portfolio_style.css
Comment (karenschulz) [reply portfolio_style.css]: I changed the font to rem, but the padding I left it in px

File: portfolio_style.css
Comment (karenschulz) [reply portfolio_style.css]: done

File: portfolio_style.css
Comment (karenschulz) [reply portfolio_style.css:121]: left it.

File: portfolio_style.css
Comment (karenschulz) [reply portfolio_style.css]: done

File: contact.html
Comment (mtmgoliath) [inline contact.html]: you need a : after the copy

File: index.html
Comment (mtmgoliath) [inline index.html]: need the : after copy again

File: portfolio.html
Comment (mtmgoliath) [inline portfolio.html]: i would probably use a h tag here for your name

File: portfolio.html
Comment (mtmgoliath) [inline portfolio.html]: : after &copy

**Review by mtmgoliath — APPROVED** [review]
great stuff, just need to update a couple of little things like : after the &copy in  each file

File: contact.html
Comment (karenschulz) [reply contact.html]: Yes. thanks for reminding me. I already changed it

File: index.html
Comment (karenschulz) [reply index.html]: it should be updated

File: portfolio.html
Comment (karenschulz) [reply portfolio.html]: hummm I will see... will play with it but if it doesnt look ok will leave.

File: contact.html
Comment (karenschulz) [reply contact.html]: Sorry... got confused... I did the : after the copy and in my browser and it looks wrong, that is why I decided to take it out. Will check other options then, because this is a strange one... Can you see the Copy symbol at all?

File: portfolio.html
Comment (karenschulz) [reply portfolio.html]: I think unless you see the actual page, it is difficult to assess

File: portfolio.html
Comment (karenschulz) [reply portfolio.html]: OK... it works with semicolon.

Comment (karenschulz) [reply portfolio.html]: At least with me, it doesn't work with : 
👍

Comment (karenschulz) [reply portfolio.html]: Problem solved

File: contact.html
Comment (mtmgoliath) [inline contact.html:70]: : not ; 
??

Comment (mtmgoliath) [reply contact.html:70]: had a look and works fine so ignore this

File: portfolio.html
Comment (mtmgoliath) [inline portfolio.html:42]: spelling mistake hamster

Comment (mtmgoliath) [reply portfolio.html:42]: also i believe it is fuchsia

**Review by mtmgoliath — APPROVED** [review]
only a couple of typos to be fixed and pushed up before you merge =]

**Pull request merged**
